README.md

```diff
@@ -1077,7 +1077,6 @@ A curated list of amazingly awesome Elixir libraries, resources, and shiny thing
 
 ## Testing
 *Libraries for testing codebases and generating test data.*
-
```

Comment:
Thanks and another thing, sorry I didn't notice before, please follow the repo description, that is `A BDD library for Elixir developers`